src/main/resources/lang/english.lang

```diff
@@ -109,6 +110,9 @@ skript command:
 			# invalid version: No changelog for the version <gold>%s<red> available
 			title: <bold><cyan>%s<reset> (%s)
 			next page: <grey>page %s of %s. Type <gold>/skript update changes %s<gray> for the next page (hint: use the up arrow key)
+	list:
+		enabled: <white>Enabled scripts: <lime green>%s
```

Comment:
Can we have tooltips here? we can display the path of the script using tooltips

Reply:
No it needs to go to console too